javascript/src/HtmlEditorField.js

```diff
@@ -844,7 +844,14 @@ $.entwine('ss', function($) {
 				action = "update";
 			}
 
-			if(href.match(/^mailto:(.*)$/)) {
+            if(href.match(/^mailto:(.*)\?subject=(.*)$/)) {
```

Comment:
I'm suggesting we make this a bit more versatile so we don't introduce regressions with future features and we allow this to be extensible by other devs that decide to add other parameters in their own code-base